config/service_providers.yml

```diff
@@ -108,6 +114,8 @@ development:
   'https://dashboard.login.gov':
     friendly_name: 'Dashboard'
     agency: 'GSA'
+    agency_id: 2
+    uuid_priority: 30
```

Comment:
The UUID priority is just for the initial migration, right?

Reply:
correct.  it simply resolves the situation where there are currently duplicates.